design/FY2024/NFP-Refrigerant-Migration-Decarbonization.md

```diff
@@ -0,0 +1,207 @@
+       \type object-list
+       \object-list UniVariateFunctions
+       \note quadratic curve = a + b * dT is typical, other univariate curves may be used
+  N10; \field Refrigerant Migration Minimum Temperature Difference
+       \type real
+       \minimum>=0
+       \default 0.0
+       \note refirgerant migration is disabled below this minimum limit and
+       \note when the load is greater than calculated using the prevoius field.
+
+Chiller:Electric,
+  A14; \field Refrigerant Migration Temperature Difference Curve Name
+       \type object-list
+       \object-list UniVariateFunctions
+       \note quadratic curve = a + b * dT is typical, other univariate curves may be used
+
+Chiller:Electric:EIR,
+  A17; \field Refrigerant Migration Temperature Difference Curve Name
+       \type object-list
+       \object-list UniVariateFunctions
+       \note quadratic curve = a + b * dT is typical, other univariate curves may be used
+
+Chiller:Electric:Reformulated:EIR,
+  A16; \field Refrigerant Migration Temperature Difference Curve Name
+       \type object-list
+       \object-list UniVariateFunctions
+       \note quadratic curve = a + b * dT is typical, other univariate curves may be used
+```
+
+### Free cooling in air-cooled chillers ###
+
+Free cooling from air-cooled chillers typically happens through air-side economizers. Although the previous discussion of refrigerant migration applied to water-cooled chillers, there is no reason this same concept could not be applied to air-cooled equipment. This concept would be applied to the same chillers described above.
+
+### Integrated water-side economizer with air-cooled chiller ###
+
+Integrated water-side economizer with air-cooled chillers is likely possible with existing objects with minor additions to control of the water flow through the WWHX (HeatExchanger:FluidToFluid). One side of the WWHX would be connected to the supply side inlet node of the chiller evaporator (or other appropriate branch in the plant, i.e., pre-cool entire plant loop) while the other side of the WWHX would be connected to a demand side branch of the condenser loop. Activation of the condenser loop flow through the HX, to activate free cooling, would be accomplished through controls modification (2). The WWHX object already has inputs for Control Type, Heat Exchanger Setpoint Node Name, Minimum Temperature Difference to Activate Heat Exchanger, Operation Minimum Temperature Limit, and Operation Maximum Temperature Limit. I am unsure at this time which of these fields would be used for configuration as an integrated water-side economizer or if other inputs would be required. It may be as simple as adding a new Control Type = IntegratedWaterSideEconomizer to allow the proper controls. Investigation into the new control will include review of the Component Override Loop inputs (e.g., using chiller condenser inlet or evaporator outlet node temperature as a control point).
+
```

Comment:
I suspect you won't need to add a new control type to the WWHX.  It can use a CoolingSetpoint* method.  The chiller in series downstream is controlled by component setpoint based operation instead of load-based dispatch and should just pick up the modified temperatures. 

CoolingSetpointOnOffWithComponentOverride is intended for the "all or nothing"  parallel configuration.  May need some testing.